### PR TITLE
Mahir branch merging into master

### PR DESCRIPTION
**Pull request by adithyabhatkajake**
Fixed all broken images and rebased with our branch.
Able to merge.

### CODE REVIEW

**Comment by adithyabhatkajake** [comment]
Shall I merge or are there any other changes? @salman-bhai @mahirjain25

**Comment by mahirjain25** [comment]
Ok so compile those with jekyll and send a PR then ?

**Comment by adithyabhatkajake** [comment]
Add the articles, compile them. Push the commits to this branch. No need to raise another PR.

**Comment by salman-bhai** [comment]
@adithyabhatkajake try to squash the commits instead of sending all of them! :smile:

**Comment by adithyabhatkajake** [comment]
Squashing and rebase have their limitations. Squashing can only be done with adjacent commits on the same branch. With branding and merge commits all over the place there are very few squash commits that can be done. Rebasing will change the history and render all the local copies useless and create problems. I'm trying my best to reduce the number of commits. But the people who are contributing must also reduce branching and the number of commits.

**Comment by adithyabhatkajake** [comment]
This is a merge nightmare. People who fork the repo keep outdated versions of the origin/master. I had to spend 2 hours fixing the commit mess.

**Comment by adithyabhatkajake** [comment]
This is not merge ready. @mahirjain25 Fix 
* the spacing between ##content to ## content. 
* Remove the angular code apprearing everywhere
* Are you the author of both the new articles? if not add the correct authors.

**Comment by mahirjain25** [comment]
Have fixed the formatting, and have removed angular code
I am waiting for usernames of the authors of the both new articles.

File: blog_src/_posts/2017-09-03-What-Makes-a-Smart-Home-Smart.md [hunk 1]
```diff
@@ -41,11 +40,11 @@ Now you want to turn one of your appliance, let's say your water heater. You wil
...
+ ![Smart Home](data:image/*;base64,/9j/4AAQSkZJRgABAQAAAQABAAD//gA+Q1JFQVRPUjogZ2QtanBlZyB2MS4wICh1c2luZyBJSkcgSlBFRyB2NjIpLCBkZWZhdWx0IHF1YWxpdHkK/9sAQwAIBgYHBgUIBwcHCQkICgwUDQwLCwwZEhMPFB0aHx4dGhwcICQuJyAiLCMcHCg3KSwwMTQ0NB8nOT04MjwuMzQy/9sAQwEJCQkMCwwYDQ0YMiEcITIyMjIyMjIyMjIyMjIyMjIyMjIyMjIyMjIyMjIyMjIyMjIyMjIyMjIyMjIyMjIyMjIy/8AAEQgDKgRDAwEiAAIRAQMRAf/EAB8AAAEFAQEBAQEBAAAAAAAAAAABAgMEBQYHCAkKC//EALUQAAIBAwMCBAMFBQQEAAABfQECAwAEEQUSITFBBhNRYQcicRQygZGhCCNCscEVUtHwJDNicoIJChYXGBkaJSYnKCkqNDU2Nzg5OkNERUZHSElKU1RVVldYWVpjZGVmZ2hpanN0dXZ3eHl6g4SFhoeIiYqSk5SVlpeYmZqio6Slpqeoqaqys7S1tre4ubrCw8TFxsfIycrS09TV1tfY2drh4uPk5ebn6Onq8fLz9PX29/j5+v/EAB8BAAMBAQEBAQEBAQEAAAAAAAABAgMEBQYHCAkKC//EALURAAIBAgQEAwQHBQQEAAECdwABAgMRBAUhMQYSQVEHYXETIjKBCBRCkaGxwQkjM1LwFWJy0QoWJDThJfEXGBkaJicoKSo1Njc4OTpDR
```
Comment (adithyabhatkajake) [inline blog_src/_posts/2017-09-03-What-Makes-a-Smart-Home-Smart.md]: Why is this binary image file present?

File: blog_src/_posts/2017-09-03-What-Makes-a-Smart-Home-Smart.md [hunk 2]
```diff
@@ -55,12 +54,12 @@ Having doubts? Let us take an example.
...
...KDPX43rf9B6/G9fQGee43y8wZ5GdgYvoNDMXseoMKBgu0CMED7JinSVxl8TFMxhw9WoxpTcxvijY72jCJ4VvKvfKJAwU0S6LrOw2zM1FZw1KM7ojYWdd8REwwyz2mYaWi6iWvolkE3bIgtzgClX1U47yx2SSQlPQvlYR6e4/2WqY2x33K8JWZNX/AEp8khT5qF80NmsSqoxC2mVl2WfW+N5/F9ejt2vS2e+VwY9T3zPo65Hl6XvvkYM9df8A5M/qWgMGQ/cCBKBn1RF8xYR5TkNfFGrNccqiGUyae8h+IjkBXflEgYH+Sgg6m5GEt/RLQJsi6Kl8BAz81OvcsdkpKwSWd8koNW0ekrd1W1EZjBT5nsarBci2ynTV0ENGvaVU48AtZleemaDlr7pVc/0l/h+mWw9Mz13zv0L65b9DzH/0L5GDCjBq01uFai+RkP35+QjnYEoXEryEKc7CMlRaXjJDtkVOruJfor6mMEfJgkwvzBdJa+EGf/Tr3LHZKUgiyvk2wp0+oxRvOSgm45ewU4a+0bhJ3XK/5Nw1b3F8kNmvaVUo9P7dQrr8rQuU/wCr3CSvu01xLK3/AKP1zvne4Mbi+R/wLi4P+AeV8vPW4Mz/AHXBi47Wf78rA9g2CUCUJB7BGqRYc8cZjrS1hCvMRInIyW1EtHE2pWhi/RW5wi/mp17ljshJWBFkRCPBcfPqNUlDPiCdJorRzVftKeJIXJv2TVffNKTXtKqkeB25eIHpeieUGhg0fTk9QWgZ1UXD/wCj9M75XG/QuNgZi4uDHpcX10M7Az0Cgf8ABmLi4PXbP/ZmewQCBGHTvYIVZIJQdPqGI/ZFOrD9MV7tHxaw8wfOoWImZarBKr7ZLXwgz81OvcsdkpTYJBJ4to9Jce3agsRe0qQZ6IuFvkkKkcWxmLi4uEINfZl1OPT+8nV56YVkcX1I+IwWgSr6pSXlyY5IvNhkuVTwZH+j9Pgeov8AA9R/9C4vbK+dgehg/qRj0vYXB6ZHoWl7glBWm1xtl/szPYN75uBJ9UEYcPqiNoQUtKe0p/8A5pVEeqBktbCeFBELBa+TBq81OvG+dkoTYIaNw+rEoxnrIbQ1G7lbxr3uFvkkLfNQuLi+SGzc7MypR6bo/OxC9M0RvkWpjhsOSMtwk/oS7BK+IMtFyif0Pp0PPoenT9Oied87j0NWo0B6Fp6eWgIW
```
Comment (adithyabhatkajake) [inline blog_src/_posts/2017-09-03-What-Makes-a-Smart-Home-Smart.md]: More Binary image files.

**Comment by salman-bhai** [comment]
Is this PR relevant?

**Comment by adithyabhatkajake** [comment]
* Add sachin, adil m rajiv, sachit krishnan and nikhil govindrajan entries to authors.yml
* Correct the dates

**Comment by salman-bhai** [comment]
@mahirjain25 , last strech :stuck_out_tongue_closed_eyes:

**Comment by mahirjain25** [comment]
I'm on it xD 
I need the right dates but.

**Comment by adithyabhatkajake** [comment]
Ask them. According to schedule we should have 3 new articles every week. So schedule accordingly. Also, I think something was posted on the group regarding schedules.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -155,7 +155,19 @@ Configuration file: none
 Create a new branch in your local repo by the command `git checkout -b <branch_name>`
 
 
+<<<<<<< HEAD
```
Comment (salman-bhai) [inline CONTRIBUTING.md]: Conflicts! :p

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -155,7 +155,19 @@ Configuration file: none
...
 Commit and push your changes to your repository. Once those changes are in, go ahead and make a pull request to the [main blog's repo](https://github.com/IEEE-NITK/ieee-nitk.github.io).
+=======
+<<<<<<< HEAD
```
Comment (salman-bhai) [inline CONTRIBUTING.md]: Conflicts! :p

File: blog_src/_posts/2017-09-04-Rock-Paper-Scissors.md [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+description: 'A deeper look into a popular childhood game'
+tags:
+- Game Theory
+- Strategy
+categories:
+- CAS
+github_username: 'sssachin300'
+---
+
+# A Nasty Game
```
Comment (salman-bhai) [inline blog_src/_posts/2017-09-04-Rock-Paper-Scissors.md]: No need for the heading, it's already there!

File: blog_src/_posts/2017-09-04-Airbus E-Fan.md [hunk 6]
```diff
@@ -0,0 +1,77 @@
...
+- Environment
+categories:
+- Piston
+github_username: 'sachitkrishnan'
+---
+
+
+
+
+# The Airbus E-Fan Project
```
Comment (salman-bhai) [inline blog_src/_posts/2017-09-04-Airbus E-Fan.md]: No need for the heading it's already there!

**Comment by salman-bhai** [comment]
@adithyabhatkajake all okay from me! Let's finish this off! :stuck_out_tongue_closed_eyes:

**Comment by adithyabhatkajake** [comment]
Dude the title and the headers were different. That's why I left them there.

**Comment by salman-bhai** [comment]
Yeah that's why it looks off!

**Comment by salman-bhai** [comment]
Let's leave it for now! If the authors have Issues, we'll tell them to change it later! Cool?

**Comment by adithyabhatkajake** [comment]
Okay. Will merge soon.

**Comment by salman-bhai** [comment]
Awesome! :stuck_out_tongue_closed_eyes: @mahirjain25 treat from  me for this :cake:

**Comment by mahirjain25** [comment]
Haha, that sounds great :smile:

**Pull request merged**